src/rendervulkan.cpp

```diff
@@ -1102,6 +1102,11 @@ static bool init_device()
 	clockFeatures.pNext = (void *) std::exchange(deviceCreateInfo.pNext, &clockFeatures);
 	clockFeatures.shaderDeviceClock = VK_TRUE;
 
+	VkPhysicalDeviceSamplerYcbcrConversionFeatures ycbcrFeatures = {};
+	ycbcrFeatures.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SAMPLER_YCBCR_CONVERSION_FEATURES;
+	ycbcrFeatures.pNext = (void *) std::exchange(deviceCreateInfo.pNext, &ycbcrFeatures);
```

Comment:
Probably makes sense to std exchange this onto device features if we have it to avoid the unnecessary cast?